extension.html

```diff
@@ -117,40 +117,101 @@ <h2>Extensions to the <code>PointerEvent</code> interface</h2>
+                <li><p> Initialize the rest of the attributes the same way as <a>PointerEvent</a>.
+              </ol>
+            </p>
+            <div class="note">
+              The PointerEvent's attributes will be initalized in a way that is best representative
+              of all event in the coalesced event list.
+              For example its <a href="https://www.w3.org/TR/pointerlock/#widl-MouseEvent-movementX">
+                movementX</a> and
+              <a href="https://www.w3.org/TR/pointerlock/#widl-MouseEvent-movementY">movementY</a>
+              ([[!POINTERLOCK]]) COULD be the sum of those of all the coalesced events.
+            </div>
+
+            <p>A <a>PointerEvent</a> has an associated <dfn> predicted event list</dfn> (a list of
+            zero or more <code>PointerEvents</code>). If this event is a <code>pointermove</code>
+            event, it is a sequence of <code>PointerEvents</code> the user agent predicts that will
+            follow the events in <a>coalesced event list</a> in the future; otherwise it is the
+            empty list.
+            The number of events in the list and how far they are from the current timestamp are
+            determined by the user agent and the prediction algorithm it uses.</p>
+
+            <p>The events in the <code> predicted event list</code> will have increasing
+            <a href="https://dom.spec.whatwg.org/#dom-event-timestamp"><code>timeStamps</code></a>
+            ([[!WHATWG-DOM]]), so the first event will have the smallest <code>timeStamp</code>
+            which should still be greater than the last event in the <a>coalesced event list</a>.
+            </p>
+
+            <p> When a <var>PointerEvent</var> is created, run the following steps for each
+            <var>event</var> in the <a> predicted event list</a>:
+              <ol>
+                <li><p>Set <var>event</var>'s <code>pointerId</code>, <code> pointerType</code>,
+                  <code>isPrimary</code> and <code>isTrusted</code> to the <var>PointerEvent</var>'s
+                  <code>pointerId</code>, <code> pointerType</code>, <code>isPrimary</code> and
+                  <code>isTrusted</code>.</li>
+                <li><p>Set <var>event</var>'s <code>cancelable</code> and <code>bubbles</code>
+                  attributes to false.</li>
+                <li><p>Set <var>event</var>'s <a>coalesced event list</a> and <a> predicted event
+                    list</a> to empty list.</li>
+                <li><p> Initialize the rest of the attributes the same way as <a>PointerEvent</a>.
+              </ol>
+            </p>
+            </p>
+
+            <p> A <a> PointerEvent</a> has an associated <dfn> coalesced events targets dirty</dfn>
+            boolean. When an event is created the attribute must be initialized to false.</p>
+            <p> A <a> PointerEvent</a> has an associated <dfn> predicted events targets dirty</dfn>
```

Comment:
You should be able to merge these paragraphs into one and just say ..." has associated coalesced events targets dirty and predicted events targets dirty bits." Also I believe you need to remove the space inside <dfn> as that will break the linking in respec.

Reply:
Done merge into 1 paragraph and space removed.